docs/index.html

```diff
@@ -55,6 +55,16 @@ <h1>To use this website, JavaScript needs to be enabled.</h1>
 
   <div class="container">
     <header class="clearfix">
+      <br>
+      <div class="darkbutton">
+        <button onclick="darkMode()" 
+        style="height: -10px";
+    >
+          <img src="/assets/img/new_moon.svg" class="image">
+         <div style="font-size:x-small; margin-top:small ;" ><b>Dark Mode</b>
+        </div>
+             </button>
+      </div>
       <h1>Devicon <span><a href="https://github.com/devicons/devicon/releases">{{ latestReleaseTagging }}</a></span></h1>
       <h2>Devicon is a set of icons representing programming languages, designing, and development tools. You can use it as a font or directly copy/paste the SVG code into your project.</h2>
       <i>*Please read the legal conditions of using the logos here at the bottom of this header</i>
```

Comment:
In line 110 and line 148, the background color needs to be updated in the CSS as well.
IIRC, the colors are defined in `script.js`, and those need to be updated as well.

![image](https://user-images.githubusercontent.com/51391473/230792516-36c14512-cf24-45cb-88d9-559fb31fc52c.png)

Reply:
> In line 110 and line 148, the background color needs to be updated in the CSS as well. IIRC, the colors are defined in `script.js`, and those need to be updated as well.
> 
> ![image](https://user-images.githubusercontent.com/51391473/230792516-36c14512-cf24-45cb-88d9-559fb31fc52c.png)

are these the only changes you need? I actually saw these just today, my apologies

Reply:
For now, yes, those are the needed changes.